.circleci/config.yml

```diff
@@ -0,0 +1,15 @@
+version: 2.1
+jobs:
+  build:
+    working_directory: ~/gocd-riemann-notifier
+    docker:
+      - image: circleci/openjdk:8-jdk-buster
+    steps:
+      - checkout
+      - restore_cache:
+          key: circleci-gocd-riemann-notifier-{{ checksum "pom.xml" }}
+      - run: mvn install
```

Comment:
This doesn't seem like it's running tests? I didn't see any test output in CircleCI.

Reply:
```
T E S T S
```
🤦‍♂️ not sure how I missed that